sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -599,7 +601,7 @@ private static LongRunningOperationStatus toLongRunningOperationStatus(String op
                 fullBackupOperation.getStartTime(), fullBackupOperation.getEndTime(),
                 fullBackupOperation.getAzureStorageBlobContainerUri());
         } else {
-            throw new IllegalArgumentException("Operation type not supported");
+            throw new UnsupportedOperationException("Operation type not supported");
```

Comment:
nit: Is it possible to show the name of the type here? Now that the exception type is `UnsupportedOperationException`, the string doesn't add any additional information.